### DISCUSSION TITLE
📢 Software License updates

### DISCUSSION THREAD

**Discussion by d2phap** [d2phap/ImageGlass#1505] [General]
Starting from **ImageGlass 8.8**, it is recommended (but not required) to register for an ImageGlass license to use the app in your organization. Please visit https://imageglass.org/license for more details.

Phap

**Comment by compuguy** [comment]
I'm a bit confused here. You're post here says that "it is required to register for an ImageGlass license to use the app in your organization". But at https://imageglass.org/license it says the following:

> 1c) If you intend to use ImageGlass Classic at your place of business or for commercial purposes, it's recommended but not enforced to register at https://imageglass.org/license.

So for commercial purposes, is it required to be licensed or is it *recommended* to register? 😕

**Reply by d2phap** [reply]
Sorry, this one is outdated as I changed to make it not required

**Reply by compuguy** [reply]
Thanks for the clarification @d2phap!

**Comment by MerlinYoda** [comment]
It would be really helpful if you could mention what purpose providing registration information serves on the license page. Seeing as the software is licensed under the GPL, it is available to anyone, even organizations, to download and compile themselves thus forgoing any need to consider registering but they could be encouraged to submit registration information if they knew what the information they provided was used for (and also that it was reasonably well guarded to address privacy concerns).

As such, you may want to consider rephrasing the applicable section in the registration form to more expressly highlight that submitting a registration entirely optional like so:

>Except for the [ImageGlass Store version](https://www.microsoft.com/store/productId/9N33VZK3C7TH), [ImageGlass](https://imageglass.org/) is provided free of charge for personal and professional use. **However, if you plan to use ImageGlass in a professional setting such as at work or in an organization, submitting an optional registration to the author of ImageGlass would be appreciated.**

In the EULA, section 1c would need to be changed as well to match:
>1.c. If you intend to use ImageGlass Classic at your place of business or for commercial purposes, submitting an optional registration at https://imageglass.org/license would be appreciated.

Additionally, the name and logo of ImageGlass are trademarks of yours so the phrasing in section 2 is much longer than it needs to be. It is well understood that open source software licenses do not generally extend to trademark usage. The Firefox browser is a good example of this. Anyone is free to modify and redistribute their own versions of the Firefox browser but they are not openly allowed to use the Firefox name or logo in doing so as they are trademarks for the Mozilla Foundation. As such you can greatly trim down section 2 as follows:

>2. The ImageGlass name and logo are trademarks of Duong Dieu Phap. Written permission must be obtained prior to using the ImageGlass name or logo in any distributed works.

**Reply by d2phap** [reply]
Hi @MerlinYoda, I have updated the license and the registration form according to your suggestion!

> It would be really helpful if you could mention what purpose providing registration information serves on the license page.

I don't know how to phrase it out but this mostly for myself to know who is using ImageGlass in their work based on the country and website provided. As ImageGlass does not collect any data, so this is the only way for my curiosity.

**Reply by MerlinYoda** [reply]
>I don't know how to phrase it out but this mostly for myself to know who is using ImageGlass in their work based on the country and website provided. As ImageGlass does not collect any data, so this is the only way for my curiosity.

OK. I think the general term for this would be "usage data". 

I know some developers of various software products will show off names of large companies that use them sometimes. Also, if they find certain types of companies often use their software then they might favor new features that would be useful in those industries over others.
